src/OrchardCore/OrchardCore.DisplayManagement/PositionWrapper.cs

```diff
@@ -19,6 +19,8 @@ public class PositionWrapper : IHtmlContent, IPositioned, IShape
 
 		public IDictionary<string, string> Attributes { get; }
 
+        public IDictionary<object, object> Properties { get; } = new Dictionary<object, object>();
```

Comment:
This should be lazy, I don't think these are accessed anywhere in a wrapper.

Reply:
Done - assuming this is what you meant by lazy
`public IDictionary<string, object> Properties => _properties = _properties ?? new Dictionary<string, object>();`